README.md

```diff
@@ -148,6 +148,52 @@ HLS features supported:
 HLS features **not** supported:
  - X-SNAP attribute in interstitials
 
+<details>
+<summary>
+<h3>Supported HLS tags</h3>
+</summary>
+
+For details on the HLS format and these tags' meanings, see https://datatracker.ietf.org/doc/html/draft-pantos-hls-rfc8216bis
+
+<h4> Multivariant Playlist tags</h4>
+
+- `#EXT-X-STREAM-INF:<attribute-list>`
+  `<URI>`
+- `#EXT-X-MEDIA:<attribute-list>`
+- `#EXT-X-IMAGE-STREAM-INF:<attribute-list>`
+- `#EXT-X-I-FRAME-STREAM-INF:<attribute-list>`
+- `#EXT-X-SESSION-DATA:<attribute-list>`
+- `#EXT-X-SESSION-KEY:<attribute-list>` EME Key-System selection and preloading
+- `#EXT-X-START:TIME-OFFSET=<n>`
+- `#EXT-X-CONTENT-STEERING:<attribute-list>` Content Steering
+- `#EXT-X-DEFINE:<attribute-list>` Variable Substitution (`NAME,VALUE,QUERYPARAM` attributes)
+
+<h4>Media Playlist tags</h4>
```

Comment:
We also support `EXT-X-PLAYLIST-TYPE`, right?

Reply:
Done!